src/NServiceBus.Core/Routing/INonCompetingConsumersTransport.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NServiceBus.Routing
+{
+    /// <summary>
+    /// Marks the transport as unable to use competing consumers pattern.
+    /// </summary>
+    public interface INonCompetingConsumersTransport
```

Comment:
I think we should remove this interface. Let's not try to put some abstraction of something which only is related to MSMQ for now. Let's introduce that when we have something else which would benefit from this?